lib/field.ts

```diff
@@ -46,12 +48,12 @@ export default class Field implements EntryFieldAPI {
     return this._getFieldLocale(locale).getValue()
   }
 
-  setValue(value: any, locale?: string) {
-    return this._getFieldLocale(locale).setValue(value)
+  async setValue(value: any, locale?: string) {
+    return await this._getFieldLocale(locale).setValue(value)
   }
 
-  removeValue(locale?: string) {
-    return this.setValue(undefined, locale)
+  async removeValue(locale?: string) {
+    await this.setValue(undefined, locale)
```

Comment:
did you mean `return await this.setValue`

Reply:
No. This is `removeValue`, so we don't want to return anything -> return `undefined`

Reply:
Aren't we changing the signature then?

Reply:
The API stays the same and the behavior is as well.
Only the types started complaining, so I changed things a bit. `setValue` returns `SerializedJSONValue | undefined` which does not match the `undefined`/`void` return value of `removeValue`. So, instead of forwarding a return value that will always be `undefined`, I now don't return anything as that's `void` by default.